spotify-app/spotify-rc-asier/scripts/main.js

```diff
@@ -0,0 +1,147 @@
+$(document).ready(function () {
+    "use strict"
+
+    $("#listArtists, #listAlbums, #listSongs, #backToArtists, #error").hide()
+
+
+    const showError = () => {
+        $("#error").empty()
+        $("#error").append('<div class="d-block bg-light p-5"><h1 class="display-1 text-success text-center" >  No artist matching your search! </h1></div>')
+
+    }
+
+    // ------- submit form and get artists, pass them to the api and show them on html -------//
+
+    $('form').submit(function (e) {
+        e.preventDefault();
+        const query = $('input').val()
+
+        $('form').trigger('reset')
+        $("#listArtists").show()
+        $("#listAlbums, #listSongs, #backToArtists, #error").hide()

```

Comment:
hey! you can use other thing.
```javascript
const $mainContainer = $('.container-fluid')
$mainContainer.childer().hiden()
```

Reply:
I could, but 'form' is a 'main' child too... So I'd had to specify all children but 'form' to hide